docs/reference-guides/theme-json-reference/theme-json-living.md

```diff
@@ -95,6 +95,7 @@ Settings related to typography.
 | customFontSize | boolean | true |  |
 | fontStyle | boolean | true |  |
 | fontWeight | boolean | true |  |
+| fluid | object |  |  |
```

Comment:
If we have `fluidSize` as an option inside the typography array, do we also need this setting? I think we can infer whether or not to use fluid typography simply from the settings in the typography array itself.

Reply:
I've made it optional, as the first place to look for min//max viewport widths

Reply:
Wondering if we should make it `__experimentalFluid`?